internal/grpcwrap/provider.go

```diff
@@ -73,6 +74,18 @@ func (p *provider) GetSchema(_ context.Context, req *tfplugin5.GetProviderSchema
 			Block:   convert.ConfigSchemaToProto(dat.Body),
 		}
 	}
+	for typ, dat := range p.schema.EphemeralResourceTypes {
+		resp.EphemeralResourceSchemas[typ] = &tfplugin5.Schema{
+			Version: int64(dat.Version),
+			Block:   convert.ConfigSchemaToProto(dat.Body),
+		}
+	}
```

Comment:
Shall we split this into another PR and target `main`? This seems merge-worthy before anything from our WIP branch reaches `main` and less related to what we're working on.

Reply:
I have raised https://github.com/hashicorp/terraform/pull/36878